inc/saiswitch.h

```diff
@@ -3069,6 +3068,42 @@ typedef enum _sai_switch_attr_t
      * @default empty
      */
     SAI_SWITCH_ATTR_SELECTIVE_COUNTER_LIST,
+    
+    /**
+     * @brief Trim packets to this size to reduce bandwidth
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 128
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_SIZE,
+
+    /**
+     * @brief New packet trimming DSCP value
+     *
+     * @type sai_uint8_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_DSCP_VALUE,
+
+    /**
+     * @brief Is the new queue index for a trimmed packet mapped from DSCP
+     *
+     * @type sai_bool_t
```

Comment:
@marian-pritsak is this a valid type?

Looks like compilation fails with:
```
ERROR: invalid enum specified 'sai_bool_t' on SAI_SWITCH_ATTR_PACKET_TRIMMING_QUEUE_INDEX_MAPPED_FROM_DSCP
ERROR: sai_bool_t == SAI_SWITCH_ATTR_PACKET_TRIMMING_QUEUE_INDEX_MAPPED_FROM_DSCP not ending on enum name BOOL
```

Are we missing a declaration?

Reply:
type should be bool